cirq-core/cirq/transformers/dynamical_decoupling.py

```diff
@@ -202,11 +325,17 @@ def add_dynamical_decoupling(
     Returns:
           A copy of the input circuit with dynamical decoupling operations.
     """
-    base_dd_sequence, pauli_map = _parse_dd_sequence(schema)
+
+    if context is not None and context.deep:
+        raise ValueError("Deep transformation is not supported.")
+
     orig_circuit = circuit.freeze()
 
-    busy_moment_range_by_qubit = _calc_busy_moment_range_of_each_qubit(orig_circuit)
+    labeled_circuit = _LabeledCircuit.from_circuit(orig_circuit, single_qubit_gate_moments_only)
+
+    print(f"Preprocessed input circuit repr:\n{repr}")
```

Comment:
there are 3 options here
1. remove this print line
2. hide it behind a `if verbose:` and set verbose to default to False
3. turn it into a logging.info call